python/pyspark/rdd.py

```diff
@@ -56,6 +56,22 @@
 __all__ = ["RDD"]
 
 
+class PythonEvalType(object):
+    """
+    Evaluation type of python rdd.
+
+    These values are internal to PySpark.
```

Comment:
it's weird to say it's internal and then ask users to use it in public udf API...